lib/rules/attribute-indentation.js

```diff
@@ -38,6 +38,17 @@ function canApplyRule(node, type, config) {
 }
 
 module.exports = class AttributeSpacing extends Rule {
+  static get meta() {
+    return {
+      description: 'requires params & attributes to be indented',
+      category: 'Stylistic Issues', // 'Deprecated Rules', 'Possible Error', 'Best Practices',
+      recommended: false,
```

Comment:
```suggestion
      presets: {},
```

in contrast to e.g.:

```js
      presets: { recommended: true, 'a11y-strict': 'strict' },
```